_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
> I see your point there. I don't want to sound too technical here while describing a feature but it's possible that fresh is not a real feature of PWA. Maybe what is a feature is the control over the life cycle.

It *is* a feature, but it isn't clear how *important* it is. I'm not sure I would give it a top-level "item" (nor what that item would be called). Nevertheless, if you'd like to do so, then here's a version of your new text with some suggested edits:

```
In the traditional web model, users are never prompted to install a web app:
as soon as they visit a web site, they get the latest version of it. Progressive Web Apps
give you complete control over the web app lifecycle, so you decide if you want
silent background updates or prefer to let users confirm them.
```


> Actually, this can be enriched by another user story:
> 
>> As a user, I prefer to keep the actual version and not to update immediately.
> Perhaps it is against other use cases but it is definitively a valid use case and very related to life-cycle customization.

Yes, it's a valid use case, but it's also an application-level use case, not a platform-level one. I.e. Progressive Web App enabling technologies like Service Workers don't provide this to users, neither at the platform nor at the browser layers.

Service Workers do make it possible for apps to provide it, but apps can provide it with existing technologies. So I wouldn't advertise this as a benefit of Progressive Web Apps.